specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-15-preview/images.json

```diff
@@ -0,0 +1,543 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure OpenAI DALL-E API version 2023-04-15-preview",
+    "description": "Cognitive Services API to support OpenAI DALL-E",
+    "version": "2023-04-15-preview"
+  },
+  "host": "eastus.api.cognitive.microsoft.com",
+  "basePath": "/dalle",
```

Comment:
Do we actually need the basePath? I could not find any for FT? @bexxx

Reply:
yes, it's openai in the parameterized host, see other comment. You add imagess collection next to finetunes, ... under openai, right?

Reply:
found the other section at the bottom. just remove this line. also pls move all this stuff to the bottom. start with paths.